src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentAddAction.java

```diff
@@ -124,7 +124,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         data.commentId = commentId;
         data.showCommentToString = joinParticipantTypes(createdComment.showCommentTo, ",");
         data.showGiverNameToString = joinParticipantTypes(createdComment.showGiverNameTo, ",");
-        
+
+        data.setCommentIds(commentId);
```

Comment:
@belyabl9, this should already be fixed in #5689, we can remove this. Not sure why the changes there did not show up here.